### PR TITLE
[Fix] Rule validation code crm_profiling. [15.0]

### PR DESCRIPTION
**Pull request by nikunjantala**

### CODE REVIEW

File: crm_profiling/__manifest__.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
     """,
     "author": "OpenERP SA",
     "website": "https://github.com/OCA/server-tools",
-    "depends": ["base", "crm", "sale"],
+    "depends": ["base", "crm", "sale", "contacts"],
```
Comment (jans23) [inline crm_profiling/__manifest__.py]: Is "sale" dependency still required?

Comment (manengel) [reply crm_profiling/__manifest__.py]: Should no longer be the case if I see correctly.

**Review by manengel — CHANGES_REQUESTED** [review]
I found this error when wanting to assign a pair of a question and an answer to a partner in the respective Profiling section of the Contacts FormView:
```
Traceback (most recent call last):
  File "/srv/odoo/odoo/parts/odoo/odoo/addons/base/models/ir_http.py", line 237, in _dispatch
    result = request.dispatch()
  File "/srv/odoo/odoo/parts/odoo/odoo/http.py", line 687, in dispatch
    result = self._call_function(**self.params)
  File "/srv/odoo/odoo/parts/odoo/odoo/http.py", line 359, in _call_function
    return checked_call(self.db, *args, **kwargs)
  File "/srv/odoo/odoo/parts/odoo/odoo/service/model.py", line 94, in wrapper
    return f(dbname, *args, **kwargs)
  File "/srv/odoo/odoo/parts/odoo/odoo/http.py", line 348, in checked_call
    result = self.endpoint(*a, **kw)
  File "/srv/odoo/odoo/parts/odoo/odoo/http.py", line 916, in __call__
    return self.method(*args, **kw)
  File "/srv/odoo/odoo/parts/odoo/odoo/http.py", line 535, in response_wrap
    response = f(*args, **kw)
  File "/srv/odoo/odoo/parts/odoo/addons/web/controllers/main.py", line 1343, in call_kw
    return self._call_kw(model, method, args, kwargs)
  File "/srv/odoo/odoo/parts/odoo/addons/web/controllers/main.py", line 1335, in _call_kw
    return call_kw(request.env[model], method, args, kwargs)
  File "/srv/odoo/odoo/parts/odoo/odoo/api.py", line 464, in call_kw
    result = _call_kw_multi(method, model, args, kwargs)
  File "/srv/odoo/odoo/parts/odoo/odoo/api.py", line 451, in _call_kw_multi
    result = method(recs, *args, **kwargs)
  File "/srv/odoo/odoo/parts/OCA/social/mass_mailing_partner/models/res_partner.py", line 74, in write
    res = super().write(vals)
  File "/srv/odoo/odoo/src/addons/crm_profiling/models/crm_profiling.py", line 117, in write
    (6, 0, self._recompute_categ(vals["answers_ids"][0][2]))
  File "/srv/odoo/odoo/src/addons/crm_profiling/models/crm_profiling.py", line 85, in _recompute_categ
    self.env.cr.execute(query)
  File "/usr/local/lib/python3.8/site-packages/decorator.py", line 232, in fun
    return caller(func, *(extras + args), **kw)
  File "/srv/odoo/odoo/parts/odoo/odoo/sql_db.py", line 90, in check
    return f(self, *args, **kwargs)
  File "/srv/odoo/odoo/parts/odoo/odoo/sql_db.py", line 313, in execute
    res = self._obj.execute(query, params)
Exception

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/srv/odoo/odoo/parts/odoo/odoo/http.py", line 643, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/srv/odoo/odoo/parts/odoo/odoo/http.py", line 301, in _handle_exception
    raise exception.with_traceback(None) from new_cause
psycopg2.errors.SyntaxError: syntax error at or near "SELECT"
LINE 5:             SELECT id, categ_id
                    ^
```
Please have a look on that. Do not know if this related to these changes.

**Comment by dsolanki-initos** [comment]
> 

@manengel Please check I've fixed the error while assigning questionnaire to partner

**Review by manengel — APPROVED** [review]
Does now look good to me!

**Comment by alihashaam** [comment]
@jans23 @nikunjantala 
since we dont use labels here, is this PR ready for review?

**Comment by jans23** [comment]
We are working on a fix for 12.0 which should be migrated eventually to this PR.

**Pull request closed without merging**